### PR TITLE
Sorted sidebar

### LINKED ISSUE #371: customize order of sidebar items
Sidebar items that have been written as pages appear in alphabetical order.

We would like to be able to determine the order, e.g., by adding an additional attribute (e.g. "order: 3") to the page meta data. 

One possibility is to add code around
https://github.com/MDAnalysis/MDAnalysis.github.io/blob/e294530773afe376124d41e1bd714ade45a6efe5/_includes/sidebar.html#L25 to sort by "order".

### PR DESCRIPTION
**Pull request by orbeckst**
- fix #371 
- add `order: N` attribute to front-matter of `layout: page` pages to have them sorted by order
- update README with instructions
- preliminary ordering (please change!)
- removed UGM 2023 from sidebar

### CODE REVIEW

**Comment by lilyminium** [comment]
Order looks pretty good -- LGTM! Thanks for fixing this @orbeckst

**Comment by orbeckst** [comment]
Thanks @lilyminium for double checking!! 

(Do we need proper approvals for merging here? In any case, I'd want to give @jennaswa an opportunity to eg change the order and anyone else to comment if they feel like).

@jennaswa could I please assign the PR to you and you either wait for me to fix things if anyone requests fixes or you merge it once it's good in your eyes? (Btw, I'd ask that this be *merged* with individual commits intact so that it's clear what's happening — I'll make sure that the history remains clean.)

File: pages/team.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: page
 title: MDAnalysis team
+order: 3
```
Comment (jennaswa) [inline pages/team.md:4]:
```suggestion
order: 4
```

**Comment by orbeckst** [comment]
Thanks @jennaswa , I rewrote the history to include your change. Should be good to merge.

**Pull request merged**